### ISSUE TITLE
[BUG] retrieve message STILL BUG

### ISSUE THREAD

**Issue by LowScarlet** [SkyCraft78/DiSky#57]
**Name:** `retrieve message`
**Versions:** `1.14`

**Syntaxes affected:**
* Expression: retrieve message`

**In-depth description of the bug:**
Code:
```
    if {{@base.name}_CACHE::SERVER::server-stats} is set:
        set {_channel} to channel with id "%discordGetserverstartschatId()%"
        retrieve message with id {{@base.name}_CACHE::SERVER::server-stats} from {_channel} and store it in {_msg}
        delete discord entity {_msg}
  ````
I've confirmed that the message ID stored in `{{@base.name}_CACHE::SERVER::server-stats}` is the same as the one in discord, but the message won't be deleted!

Sky, please don't joke this time, I'm serious

**Comment by SkyCraft78** [comment]
Did you debug everything? Does the bot which executes the code have the right to delete the message ?

**Comment by LowScarlet** [comment]
Yes I have debug it with broadcast the {_msg} with output M:Ender dragon#4857:(854716431966011413)

**Comment by LowScarlet** [comment]
`bot which executes the code have` yes Ender dragon 
![image](https://user-images.githubusercontent.com/64282946/122230229-a4a35e00-cee3-11eb-80e7-17047ba0fe69.png)

**Comment by SkyCraft78** [comment]
fixed in 2.0
